loader/test/support/smart-scheduling-links.js

```diff
@@ -0,0 +1,149 @@
+const {
+  scheduleReference,
+  locationReference,
+} = require("../../src/smart-scheduling-links");
+
+/**
+ * Create a new location object with dummy location data. Matches the format
+ * returned by `getLocations()` in the smart-scheduling-links module.
+ *
+ * Optionally pass in an object with any key/value pairs to override on the
+ * location. If it has an array for the `schedules` property, each element will
+ * be used as a similar set of overrides on a dummy schedule attached to the
+ * location. Similarly, if a schedules override has an array for its `slots`
+ * property, each element will be used as a set of overrides on a dummy slot
+ * attached to the schedule.
+ *
+ * By default, each location has one schedule and each schedule has one slot.
+ * You can pass empty arrays for `schedules` (or a schedule's `slots`) to not
+ * create any. If the array has > 1 item, then the same number of
+ * scheudles/slots will be created.
+ * @param {any} [overrides]
+ * @returns {{location: any, schedules any[], slots: any[]}}
+ */
+function createSmartLocation(overrides) {
```

Comment:
This thing is kind of crazy, but I felt like it was worthwhile to make some of the tests a little more straightforward to read (and write).

Reply:
Yeah, lots of code here, but I see how it's useful!